articles/connections/database/mysql.md

```diff
@@ -72,6 +72,15 @@ function login (email, password, callback) {
 
 This script connects to a **MySQL** database and executes a query to retrieve the first user with `email == user.email`. With the `bcrypt.compareSync` method, it then validates that the passwords match, and if successful, returns an object containing the user profile information including `id`, `nickname`, and `email`. This script assumes that you have a `users` table containing these columns. You can tweak this script in the editor to adjust it to your own requirements.
 
+For the above example, the following fields must be in your custom database's `users` table:
+
+| Field | Required |
+| --- | --- | 
+| id | Yes |
+| nickname | Yes |
+| email | Yes |
+| password | Yes |
```

Comment:
What are the standard optional fields (i.e. `given_name`, etc.) ? We should put an description of each.